dashboard/app/controllers/projects_lists_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProjectsListsController < ApplicationController
```

Comment:
In that case, the `index` action doesn't seem appropriate. If `index` is all projects for a section, what will we call all projects for a student? 

You have several choices:
 1. You could overload `index` and filter with params such that it returns all projects for a section and/or student. 
 2. You could rename the actions, instead of a generic index, provide 2 appropriate actions
 3. 2 controllers as you suggested

This also brings up another question. Why does this need to be its own controller instead of new actions on the existing `ProjectsController`? Are you separating it because it's a different view and different context?